travis/tests-unit.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 #set -e
 
-if [ $TRAVIS_PULL_REQUEST == "false" ]; then
+UNIT_TESTS_ALWAYS=1
```

Comment:
Yeah I can go either way on this. On one hand I thought this switch last year seemed worthwhile even though it broke coveralls for PRs (but not for commits). But I can see the reason to want to keep that meaningful at the expense of build time. My suggestion, if we revert back to the original as Ilya proposes, would be to reverse the order so that the unit tests fire after the integration tests so that at least we can hit failures at the same speed as before (and Travis turns red and sends email notifications on the first fail, it doesn't wait for the build to finish).